website/content/guides/1.7-template-upgrade.mdx

```diff
@@ -0,0 +1,154 @@
+---
+page_title: Upgrading your template to use Packer init
+sidebar_title: Upgrade Your Template to use Packer init
+---
+
+# Upgrading your template to be compatible with `packer init`
+
+The `packer init` command is introduced in version v1.7.0 at the same time the [Packer Plugin SDK](https://github.com/hashicorp/packer-plugin-sdk)
+is officially released. The Packer Plugin SDK is a change that benefits plugins maintainers, whilst the `init` command
+benefits Packer users by providing a better control of plugin usage within a Packer template configuration.
+
+In short words, `packer init` will look at the `required_plugins` block definition within a template, take the required
+plugins specifications and download the correct binaries for that template. For details about how the command works and
+where the plugins will be installed, please refer to the [`packer init`](/docs/commands/init) documentation.
+
+-> **Note:** `packer init` does not work with legacy JSON templates. You can
+upgrade your JSON config files to HCL using the [hcl2_upgrade](/docs/commands/hcl2_upgrade) command.
```

Comment:
```suggestion
-> **Note:** `packer init` is only supported for HCL templates. You can
upgrade legacy JSON templates to HCL using the [hcl2_upgrade](/docs/commands/hcl2_upgrade) command.
```

Nit: I find positives statements easier to process.

Reply:
Oh yeah, sounds much better!